caffe2/CMakeLists.txt

```diff
@@ -534,6 +534,8 @@ if(NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
   # Required workaround for LLVM 9 includes.
   if(NOT MSVC)
     set_source_files_properties(${TORCH_SRC_DIR}/csrc/jit/tensorexpr/llvm_jit.cpp PROPERTIES COMPILE_FLAGS -Wno-noexcept-type)
+    # Force -Werror on several files
+    set_source_files_properties(${CMAKE_CURRENT_LIST_DIR}/../aten/src/ATen/native/mkldnn/Pooling.cpp PROPERTIES COMPILE_FLAGS "-Werror")
```

Comment:
Do we need to gate all of these type of additions behind `if (NOT MSVC)` + have a separate thing that adds MSVC's `-Werror` equivalent which appears to be `/WX`? (not saying we should do this here or even worry about Windows at all, just wondering what it would look like if we did)

Reply:
Good idea, but perhaps as a next step, since VC++ raises many warnings that are not considered incorrect code even by latest clang and gcc and that will make detecting them during local development cycle much harder.